sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -126,6 +127,16 @@ def foo_pipeline(tag: str, pull_image_policy: str):
     self.artifact_location = artifact_location
     return self
 
+  def set_image_pull_policy(self, policy: str):
+    """Configures the default image pull policy
+
+    Args:
+      policy: the pull policy, has to be one of: Always, Never, IfNotPresent. 
+      For more info: https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1Container.md
```

Comment:
Perhaps use a permalink instead to prevent broken link?

Reply:
https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1Container.md#v1container
```suggestion
      For more info: https://github.com/kubernetes-client/python/blob/10a7f95435c0b94a6d949ba98375f8cc85a70e5a/kubernetes/docs/V1Container.md
```